packages/manager/apps/pci-public-ip/src/hooks/order/useStore.ts

```diff
@@ -78,5 +83,9 @@ export const useOrderStore = create<Store>()((set) => {
         ...state,
         floatingIpCreation: true,
       })),
+    reset: () =>
+      set({
+        ...initialState,
+      }),
```

Comment:
You don't need to spread initial state
```suggestion
 reset: () => set(initialState),
```